sotodlib/tod_ops/filters.py

```diff
@@ -298,15 +298,28 @@ def apply(self, freqs, tod, target):
 # Filtering Functions
 #################
 @fft_filter
-def counter_1_over_f(freqs, tod, fk, n):
+def counter_1_over_f(freqs, tod, fk=None, n=None, noise_fit_stats=None):
     """
     Counter 1/f filter for noise w/ PSD that follows:
     
     w*(1 + (fk/f)**n) 
     where w is the white noise level, fk is the knee frequency, and
     n is the 1/f index.
     """
-    return 1/(1+(fk/freqs)**n)
+    if np.isscalar(fk) and np.isscalar(n):
+        return 1/(1+(fk/freqs)**n)
+
+    if fk is None and n is None and noise_fit_stats is not None:
```

Comment:
I think if you should raise an error if all 3 (`fk`, `n` and `noise_fit_stats`) are `None` at this point in the code and describe this combination of inputs in the docstring.